docs/install.md

```diff
@@ -26,6 +26,8 @@ global:
     backendUrl: "http://codacy.mycompany.com" # This value is important for VCS configuration and badges to work
   play:
     cryptoSecret: "PLEASE_CHANGE_ME" # Generate one with `openssl rand -base64 32 | tr -dc 'a-zA-Z0-9'`
+  akka:
+    sessionSecret: "PLEASE_CHANGE_ME" # Generate one with `openssl rand -base64 32 | tr -dc 'a-zA-Z0-9'`
```

Comment:
why we are indicating a different why of obtaining the secret other than: `cat /dev/urandom`
can we try to be consistent?

Reply:
You are right.. can you change all these `openssl` ?

Reply:
done